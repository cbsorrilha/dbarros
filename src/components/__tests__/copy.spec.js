import React from "react"
import renderer from "react-test-renderer"

import Copy from "../copy"

describe("Copy", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<Copy year="2020" />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
