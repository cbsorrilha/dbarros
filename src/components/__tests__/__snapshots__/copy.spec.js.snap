// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Copy renders correctly 1`] = `
<div
  className="sc-AykKC gRVqsY"
>
  © 
  2020
  , Built with
   
  <a
    href="https://www.gatsbyjs.org"
  >
    Gatsby
  </a>
   
  and ♥
</div>
`;
